doc/content/lorawan/regional-limitations-of-rf-use/_index.md

```diff
@@ -0,0 +1,65 @@
+
+# Europe (863-870 MHz)
+
+In Europe, LoRaWAN uses a range of **24 to 80 channels** with either 125 kHz or 250 kHz bandwidth, and 250 kHz bandwidth is only applied with DR (Data Rate) 6.
+
+All LoRaWAN end devices are required to operate at least on channels 868.10 MHz, 868.30 MHz, and 868.50 MHz. These channels must also have a bandwidth of 125 kHz at DR0 to DR5 and be restricted to a maximum [duty cycle](/lorawan/duty-cycle/) of 1%. Gateways must be listening on one or more of these channels. The same channels are used to send the join-request message and in that case, the duty cycle should be as follows:
+
+- Aggregated during the first hour following power-up or reset: Transmit time < 36Sec
+- Aggregated during the next 10 hours: Transmit time < 36Sec
+- After the first 11 hours, aggregated over 24h: Transmit time < 8.7Sec per 24h
+
+The [LoRa Alliance®](https://lora-alliance.org/) recommends a duty cycle limitation of 1% in the European band which means that a device can transmit for no more than 1% of the time while ensuring the maximum EIRP (Effective Isotropic Radiated Power) of +16 dBm. The purpose of the duty cycle limitation is to ensure that devices operating in the ISM band do not cause harmful interference to other devices operating in the same band. 
+
+The EIRP of 16 dBm is roughly equivalent to 40 mW and is the same as an ERP of 14 dBm, which is approximately equivalent to 25 mW. To learn more about EIRP and ERP, you can refer to the [LoRaWAN®](/lorawan/) section of The Things Network.
+
+However, the [ETSI](https://www.etsi.org/) (European Telecommunications Standards Institute) has segmented the European ISM frequency band into 6 sub-bands (K, L, M, N, P, Q) and imposes additional restrictions on the duty cycle and maximum ERP for some sub-bands (Please see pages 21-22 of the [ETSI EN300.220 V3.2.1 (2018-06)](https://www.etsi.org/deliver/etsi_en/300200_300299/30022002/03.02.01_60/en_30022002v030201p.pdf)) document.
+
+- K (863 MHz - 865 MHz): 0.1%, 25 mW ERP
+- L (865 MHz - 868 MHz): 1%, 25 mW ERP
+- M (868 MHz - 868.6 MHz): 1%, 25 mW ERP
+- N (868.7 MHz - 869.2 MHz): 0.1%, 25 mW ERP
+- P (869.4 MHz - 869.65 MHz): 10%, 500 mW ERP
+- Q (869.7 MHz - 870 MHz): 1%, 25 mW ERP
+
+For example, devices operating in the 869,4-869,65 MHz range, which is sub-band P, have a duty cycle limitation of 10%, which means that a device can transmit for no more than 10% of the time.
+
+Usually, the limitations of sub-band K, L, M, N and Q are applied for uplink transmissions and the limitations of sub-band P are mainly applied for downlink transmissions. LoRaWAN's RX2 channel is mainly used for downlink communication because there are more end devices with individual duty cycles compared to gateways, and the RX2 channel is in a specific frequency sub-band for gateways to handle the imbalance between end devices and gateways.
```

Comment:
> LoRaWAN's RX2 channel is mainly used for downlink communication

Well, RX2 is by definition downlink. So maybe rephrase this. Something like: "There are normally many more end device thatn gateways on a LoRaWAN network. Gateways can therefore run into their duty cycle limit quicker than end devices. LoRaWAN's RX2 channel is normally chose to fall in sub-bad P to benefit from the higher duty cycle limit, and provide 10x more downlinks to gateways than to end devices to combat this imbalance."